### PR TITLE
openbsd update deprecate few constants

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

File: src/unix/bsd/netbsdlike/openbsd/mod.rs [hunk 1]
```diff
@@ -936,6 +936,9 @@ pub const NET_RT_STATS: ::c_int = 4;
 pub const NET_RT_TABLE: ::c_int = 5;
 pub const NET_RT_IFNAMES: ::c_int = 6;
 #[doc(hidden)]
+#[deprecated(
+    since = "0.2.95",
+    note = "Possibily increasing over the releases and might not be so used in the field")]
```
Comment (JohnTitor) [inline src/unix/bsd/netbsdlike/openbsd/mod.rs:941]:
```suggestion
    note = "Possibly increasing over the releases and might not be so used in the field")]
```

File: src/unix/bsd/netbsdlike/openbsd/mod.rs [hunk 2]
```diff
@@ -1272,6 +1275,9 @@ pub const KERN_AUDIO: ::c_int = 84;
 pub const KERN_CPUSTATS: ::c_int = 85;
 pub const KERN_PFSTATUS: ::c_int = 86;
 pub const KERN_TIMEOUT_STATS: ::c_int = 87;
+#[deprecated(
+    since = "0.2.95",
+    note = "Possibily increasing over the releases and might not be so used in the field")]
```
Comment (JohnTitor) [inline src/unix/bsd/netbsdlike/openbsd/mod.rs:1280]:
```suggestion
    note = "Possibly increasing over the releases and might not be so used in the field")]
```

**Review by JohnTitor — CHANGES_REQUESTED** [review]
There're some typos found by @semarie.

**Comment by semarie** [comment]
with typo changes, LGTM

**Review by JohnTitor — APPROVED** [review]

**Pull request merged**